src/ui/styles/styles.scss

```diff
@@ -31,3 +32,15 @@ $govuk-new-link-styles: true;
 .ukef-multi-policy-length-hint {
   max-width: 97%;
 }
+
+.input-width {
+  width: 50%;
+}
+
+.companies-house-search {
+  flex: 1;
```

Comment:
Need to add `-moz` and `-webkit` properties as well for uniform browser results.

Reply:
display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;

Reply:
done